atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraSweepProgressStoreIntegrationTest.java

```diff
@@ -24,14 +24,16 @@
 import com.palantir.atlasdb.sweep.progress.AbstractSweepProgressStoreTest;
 
 public class CassandraSweepProgressStoreIntegrationTest extends AbstractSweepProgressStoreTest {
+    private static final CassandraContainer container = new CassandraContainer();
+
     @ClassRule
     public static final Containers CONTAINERS = new Containers(CassandraBackgroundSweeperIntegrationTest.class)
-            .with(new CassandraContainer());
+            .with(container);
 
     @Override
     protected KeyValueService getKeyValueService() {
         return CassandraKeyValueServiceImpl.createForTesting(
-                CassandraContainer.KVS_CONFIG,
+                container.getConfig(),
```

Comment:
this code snippet is the same in all of these tests. Have you thought about just making a new CassandraContainers rule which wraps the Containers rule and just contains a CassandraKeyValueServiceImpl?